client/state/action-types.js

```diff
@@ -193,6 +193,7 @@ export const LAYOUT_FOCUS_SET = 'LAYOUT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_ACTIVATE = 'LAYOUT_NEXT_FOCUS_ACTIVATE';
 export const MEDIA_MODAL_VIEW_SET = 'MEDIA_MODAL_VIEW_SET';
+export const NOTICE_CLEAR = 'NOTICE_CLEAR';
```

Comment:
Should this be named `NOTICES_CLEAR` ? In the other case, they're actions related to a single notice, but for this, it's all notices (plural).